src/test/java/com/glovoapp/planningpoker/e2e/voting/AverageTest.java

```diff
@@ -0,0 +1,70 @@
+package com.glovoapp.planningpoker.e2e.voting;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.Optional;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+
+class AverageTest {
+
+    private final WebDriver driver = getDriver();
+    private final WebDriver driver2 = getDriver();
+
+    MainPage firstUserMainPage = new MainPage(driver);
+    SessionPage firstUserSessionPage = new SessionPage(driver);
+    MainPage secondUserMainPage = new MainPage(driver2);
+    SessionPage secondUserSessionPage = new SessionPage(driver2);
+
+    private final String sessionName = "TEST222";
```

Comment:
Constant should be `static`.

```suggestion
    private static final String SESSION_NAME = "TEST222";
```